framework/bootstrap.go

```diff
@@ -216,13 +234,18 @@ func (f *framework) watchMeta(linkType string, taskIDs []uint64) {
 			}
 		}
 
+		f.log.Println("Watch ", taskID, " with path ", watchPath)
+
 		// Need to pass in taskID to make it work. Didn't know why.
 		err := etcdutil.WatchMeta(f.etcdClient, taskID, watchPath, stop, responseHandler)
```

Comment:
From the logs in the [current integration test](https://github.com/BaiGang/taskgraph/blob/pgmethod/example/bwmf/bwmf_task_test.go), it seems we actually get stuck here.